released/discovery_center/mission_3239/sample.tfvars

```diff
@@ -0,0 +1,31 @@
+#################################
+# IDP configuration
+#################################
+custom_idp                     = ""
+
+#################################cd
+# Account settings
+#################################
+globalaccount                  = "<your global account id>"
+region                         = "us10"
+subaccount_name                = "Mission-3239"
+org                            = "team1"
+cf_space                       = "dev"
+cf_url                         = "https://api.cf.us10.hana.ondemand.com"
```

Comment:
Might not work for customers automatically landing on extension landscapes. I would recommend to add a comment here that this might need adjustments